tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -65,3 +65,24 @@ Feature: State of the quota
     And user "Alice" has uploaded a file inside space "Project Delta" with content "7 bytes" to "test.txt"
     When user "Alice" uploads a file inside space "Project Delta" with content "00011 bytes" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "507"
+
+
+  Scenario Outline: Check the relative amount of quota of personal space
+    Given user "Admin" has changed the quota of the personal space of "Alice Hansen" space to "10000"
+    And user "Alice" has uploaded file "<file_upload>" to "/demo.txt"
+    When the user "Alice" requests these endpoints with "GET" with basic auth
+      | endpoint    |
+      | <end_point> |
+    Then the HTTP status code should be "200"
+    And the json responded should contain these key and value pairs:
+      | key              | value            |
+      | quota@@@free     | <quota_free>     |
+      | quota@@@used     | <quota_used>     |
+      | quota@@@total    | <quota_total>    |
+      | quota@@@relative | <quota_relative> |
+    Examples:
+      | file_upload                   | end_point                                 | quota_free | quota_used | quota_total | quota_relative |
+      | /filesForUpload/lorem.txt     | /ocs/v1.php/cloud/users/Alice?format=json | 9301       | 699        | 10000       | 6.99           |
```

Comment:
Could do similar to the following without specifying `format` query.
```suggestion
      | /filesForUpload/lorem.txt     | /ocs/v1.php/cloud/users/Alice| 9301       | 699        | 10000       | 6.99           |
```
https://github.com/owncloud/ocis/blob/486453b4c91deec3f91a501a4ff4450c2da0e1d6/tests/acceptance/features/bootstrap/OCSContext.php#L933-L941